### ISSUE TITLE
 Element/seg/attributes : blank page

### ISSUE THREAD

**Issue by RomaneMarlhoux** [TEIC/romajs#167]
A blank page is displayed when I try to specify attributes about an element (seg for exemple).

**Comment by raffazizzi** [comment]
Dear @RomaneMarlhoux can you give me more information, listing all the steps you are teking? I just tried this on roma.tei-c.org and it worked as expected:

* Start a customization from Tei All
* Went to seg -> attributes
* Changed xml:id from option to required
* Exported ODD
* Exported RNG

**Comment by RomaneMarlhoux** [comment]
Dear @raffazizzi, thank you for your answer. The problem occurs when I try to customise an ODD that I have already started working on (upload ODD). To test this, I tried creating a new one. When I create an ODD with TEI all, it works, but when I want to start from a TEI Minimal ODD, it's the same problem as with my ODD that I've already created. I hope it make sense. Thank you very much for your help !
